.github/workflows/acceptance_tests_lite.yaml

```diff
@@ -0,0 +1,41 @@
+name: Terraform Provider Acceptance Tests Lite
+on:
+  #Every day at 08:00 UTC
```

Comment:
Don't you want to run on push instead? (or do you want to make sure those tests always run because VMC can break Terraform any time?)

Reply:
On line 6 I've added a run on push, but if there is an extended period of time without pushes, then considering how cheap and fast it is to run E2E tests with ZEROCLOUD I'd prefer to have an early warning if API incompatibility between the Terraform provider and the VMC service.